README.md

```diff
@@ -91,6 +83,32 @@ Host Docker storage drivers of `overlay` and `aufs` are supported automatically.
     vagrant ssh
     ```
 
+1. Configure DC/OS Docker:
+
+    Generate `make-config.mk` automatically, interactively, or manually.
+
+    **Automatic Mode:**
+
+    ```console
+    ./configure --auto
+    ```
+
+    **Interactive Mode:**
+
+    ```console
+    ./configure
+    ```
+
+    **Manual Mode (example):**
+
+    ```console
+    cat > make-config.mk << EOM
+    MASTERS := 3
+    EOM
+    ```
+
+    See [make-defaults.mk](make-defaults.mk) for a full list of manually configurable options.
```

Comment:
Yeah, I know how make works, I just wanted to pull out the ones that are likely to be overwritten by users and provide a way to do that more persistently. You can of course still overwrite them the traditional way, at runtime.

If you'd like MASTER_MOUNTS in `make-defaults.mk` I can move it there. Are there any others?